crates/task-impls/src/vid.rs

```diff
@@ -86,7 +88,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> VidTaskState<TYPES, I> {
                         builder_commitment,
                         metadata.clone(),
                         *view_number,
-                        fee.clone(),
+                        bid_fees[0].clone(),
```

Comment:
I don't have a strong opinion, but could we use something like https://docs.rs/vec1/latest/vec1/struct.Vec1.html#method.first instead of directly indexing?

Reply:
Gah! I didn't know this was a thing!